rust/ittapi-sys/README.md

```diff
@@ -14,7 +14,8 @@ Profiling API. More details about `ittapi` are available on its [README].
 [README]: https://github.com/intel/ittapi#readme
 
 > IMPORTANT NOTE: this crate is currently only tested on Linux, macOS, and Windows platforms but
-> support for other platforms is intended; contributions are welcome!
+> support for other platforms is intended; contributions are welcome! FreeBSD is supported but
+> untested.
```

Comment:
Not sure if you intended for this newline but I guess it doesn't matter in markup.

Reply:
Yeah, it's fine.